plugin.video.ad6/addon.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="plugin.video.ad6" name="Alpe d&#39;HuZes" version="0.1.0" provider-name="Menco Bolt">
+    <requires>
+        <import addon="xbmc.python" version="2.25.0"/>
+        <import addon="script.module.routing" version="0.2.1"/>
+

```

Comment:
please add youtube as a dependency. If a user installs your addon on a fresh kodi installation and opens a directory in which you call plugin://plugin.video.youtube it will either throw an error or do nothing

Reply:
Done. Although in my own setup kodi asks to install youtube when a youtube stream is selected.